Documentation/gettingstarted/kind.rst

```diff
@@ -44,7 +45,20 @@ Install Cilium release via Helm:
       --set global.externalIPs.enabled=true \\
       --set global.nodePort.enabled=true \\
       --set global.hostPort.enabled=true \\
-      --set global.pullPolicy=IfNotPresent
+      --set global.pullPolicy=IfNotPresent \\
+      --set config.ipam=kubernetes
+
+.. tip::
+   If your local network subnet conflicts with the one picked up by kind,
+   update the ``networking`` section of the kind configuration file to specify
+   different subnets (pick a network range that does not conflict with yours)
+   and re-create the cluster.
+
+   .. parsed-literal::
+        networking:
+          disableDefaultCNI: true
+          podSubnet: "10.244.0.0/16"
+          serviceSubnet: "10.245.0.0/16"
```

Comment:
Would be nice to determine whether it's appropriate to refactor this kind setup into a dedicated file shared across the guides so we don't have to copy+paste similar instructions like this.

Reply:
Yes, that's a good point. I'll do this as part of another PR.

Reply:
See #12210.